test/mpi/main.cc

```diff
@@ -124,8 +126,6 @@ static ucc_coll_type_t coll_str_to_type(std::string coll)
         return UCC_COLL_TYPE_ALLTOALLV;
     } else if (coll == "reduce_scatter") {
         return UCC_COLL_TYPE_REDUCE_SCATTER;
-    } else if (coll == "reduce_scatterv") {
```

Comment:
looks like RSv is getting  removed by this PR (probably some incorrect rebase) ?